src/server/models/list.ts

```diff
@@ -61,7 +61,8 @@ export async function findListByAnnualReviewDate(annualReviewStartDate: Date): P
               some: {
                 type: "PUBLISHED",
                 time: {
-                  lte: subMonths(Date.now(), 1),
+                  gte: annualReviewStartDate,
```

Comment:
this needs to be adjusted.  It is intended to include items that were published since the lastAnnualReviewStartDate  OR where lastAnnualReviewDate is null (never had an annual review before)